kedro-telemetry/kedro_telemetry/masking.py

```diff
@@ -81,16 +81,19 @@ def _get_cli_structure(
     return output
 
 
-def _mask_kedro_cli(
-    cli_struct: dict[str | None, Any], command_args: list[str]
-) -> list[str]:
+def _mask_kedro_cli(KedroCLI, command_args: list[str]) -> list[str]:
```

Comment:
Is this a variable name or a actual class?

Reply:
variable name, updated it